concepts/numbers/about.md

```diff
@@ -0,0 +1,176 @@
+0.2             --> 0.2
+0.3e4           --> 2000.0
+4.0e-3          --> 0.004
+5.0e+20         --> 5e+20
+```
+
+A number with a decimal point or an exponent is considered a float and without is treated as an interger, but both have the type 'number'.
+```lua
+type(1)         --> number
+type(2.0)       --> number
+```
+
+If we want to distinguish between floats and intergers, the `type` function of the `math` module  can be used.
+```lua
+math.type(1)    --> integer
+math.type(2.0)  --> float
+```
+
+## Comparsion
+
+Integers and floats with the same value compare as equal:
+```lua
+1 == 1.0        --> true
+-2 == -2.0      --> true
+0.3e3 == 300    --> true
+```
+
+## Arithmetic
+
+Lua supports the usual set of arithmetic operators: additon, subtraction, muliplication, divison and negation. Additional it supports floor division, modulo and exponentiation.
+
+### Addition, substraction, multiplication and negation
+
+If both operands are integer the result of the operations is an integer. The same applies to floats.
+```lua
+1 + 2     --> 3
+1.0 + 2.0 --> 3.0
+```
+
+If one operand is a float, the remaining integers will be converted to floats as well and the operations results in a float.
+```lua
+1.0 + 2   --> 3.0
+```
+
+### Division
+
+Division always operates on floats and gives floats as a result.
+```lua
+1.0 / 2.0 --> 0.5
+2 / 1     --> 2.0
+```
+
+Dividing by zero will result in `inf` or `-inf`. Dividing zero by zero will result in `-nan`.
+```lua
+1 / 0     --> inf
+-1 / 0    --> -inf
+0 / 0     --> -nan
+```
+
+### Floor Divison
+
+If an integer result is needed, floor division can be used, which always rounds the quotient towards minus infinity, ensuring an integral result for all operands.
+
+~~~exercism/note
+Before version 5.3, Lua had no arithmetic operator for floor division.
+~~~
+
+```lua
+3 // 2    --> 1
+6 // 2    --> 3
+```
+
+Floor divison follows the same rules of the addition, substraction, multiplication and negation operators - if both operators are integers the result of the operation is an integer, otherwise the result is a float.
+
+```lua
+3.0 // 2   --> 1.0
+6.0 // 2.0 --> 3.0
+```
+
+When dividing by zero using floor division, Lua will raise an error.
+```lua
+3 // 0     --> stdin:1: attempt to divide by zero
+```
+
+### Modulus
+
+Modulus is used to get the remainder of the division of two operands.
+```lua
+8 % 2      --> 0
+13 % 9     --> 4
+0.3 % 0.2  --> 0.1
+```
+
+### Exponentiation
+
+Like division, exponentiation also operates on floats and gives floats as results.
+```lua
+10 ^ 2    --> 100.0
+```
+
+## Precedence
+
+All operations are evaluated according to the [operator precedence](https://www.lua.org/manual/5.4/manual.html#3.4.8). Parentheses can be used to change the precedences of an expression.
+
+```lua
+1 + 2 * 3   --> 7
+(1 + 2) * 3 --> 9
+```
+
+## Rounding
+
+The `math` module provides three rounding functions: `floor`, `ceil` rounds towards plus infinite and `modf` rounds towards zero.
```

Comment:
```suggestion
The `math` module provides three rounding functions: `floor` rounds "up" (toward negative infinity), `ceil` rounds "down" (towards negative infinity) and `modf` rounds towards zero.
```

Reply:
According to PiL (fourth edition) `ceil` rounds towards plus infinity

Reply:
Ah just saw two suggestion below you wrote "postive infinity", so this is just a typo here I guess :)

Reply:
Yep, sorry that was a typo. I meant to change to positive infinity. If PiL uses "plus infinity" that is fine with me.

Reply:
I applied your suggestion with positive and negative :)